### PR TITLE
Fix some lint errors

### PR DESCRIPTION
**Pull request by YagoGG**
Fixes some lint errors reported by JSHint.

### CODE REVIEW

File: javascripts/content_scripts/posting.floating.js [hunk 1]
```diff
@@ -183,7 +183,6 @@ if (SeamlessPosting === undefined) {
     var node = this.getNode();
     node.style.transition = 'none'; // remove transition
     this.updateVisibility();  // reset animation to the hidden state
-    node.offsetWidth;   // force reflow
```
Comment (smcgregor) [inline javascripts/content_scripts/posting.floating.js]: This is a semantic change that would require extensive testing. It is probably best to not include it here.

**Comment by smcgregor** [comment]
This looks good, with the exception of the one inline comment. Thanks for this (and other) lintings!

**Comment by YagoGG** [comment]
@smcgregor I just placed back that line.
No problem, hope they help (even just a little) ;)

Congrats on the code, BTW, it's quite clean!

**Comment by smcgregor** [comment]
Thanks!

**Pull request merged**